src/api/delivery-needed/index.js

```diff
@@ -16,12 +17,13 @@ exports.deliveryNeededRequestHandler = async (req, res) => {
     crossStreetFirst,
     crossStreetSecond,
     meta,
-    neighborhoodAreaSeeMap
+    neighborhoodAreaSeeMap,
```

Comment:
I think this got renamed a few days ago: https://github.com/crownheightsaid/mutual-aid-app/pull/64/files#diff-91728e8556239a5cf5b437773a0d7b9dR68 though it looks like it ends up not getting used after being put into the response?